pkgs/applications/science/molecular-dynamics/gromacs/default.nix

```diff
@@ -72,7 +72,7 @@ in stdenv.mkDerivation rec {
 
   meta = with lib; {
     homepage = "https://www.gromacs.org";
-    license = licenses.gpl2;
```

Comment:
1. Squash the two commits into only one
2. Reword the commit:

```
gromacs: update meta

- license is now lgpl21Plus
- accurate longDescription
```

Reply:
Done